frontend/src/components/AddMember/index.js

```diff
@@ -1,168 +1,243 @@
+      },
+      {
+        value: "4",
+        text: "Altă situație"
+      }
     ]
   };
 
-  const toggleSex = event => {
-    setFemale(event.target.value === "Feminin");
-  };
-
-  const togglePreexistingConditions = value => {
-    setPreexistingConditions(value === "Da");
-  };
+  const [currentStep, setCurrentStep] = useState(0);
+  const [userData, setUserData] = useState({
+    name: "",
+    surname: "",
+    phoneNumber: "",
+    relationType: 0,
+    age: 0,
+    gender: 0,
+    smoker: false,
+    preexistingMedicalCondition: "",
+    livesWithOthers: false,
+    quarantineStatus: 0,
+    quarantineStatusOther: 0
+  });
 
-  const toggleDisabilities = value => {
-    setDisabilities(value === "Da");
+  const setUserDataField = (field, value) => {
+    setUserData({
+      ...userData,
+      [field]: value
+    });
   };
 
-  const validateAge = event => {
-    const value = parseInt(event.target.value.replace(/[^0-9]+/g, ""));
-    // age range is between 0 and 110 - might be worth reconsidering?
-    const boundedValue = Math.min(110, Math.max(0, value));
+  const personalDataForm = (
+    <>
+      <ListHeader title="I. Date personale"></ListHeader>
+      <div className="user-profile-form-container">
+        <Input
+          type="text"
+          label={"Nume"}
+          usePlaceholder
+          defaultValue={userData.name}
+          onChange={({ currentTarget: { value } }) => {
+            setUserDataField("name", value);
+          }}
+        />
+        <Input
+          type="text"
+          label={"Prenume"}
+          usePlaceholder
+          defaultValue={userData.surname}
+          onChange={({ currentTarget: { value } }) => {
+            setUserDataField("surname", value);
+          }}
+        />
+        <Input
+          type="text"
+          label={"Număr de telefon"}
+          usePlaceholder
+          defaultValue={userData.phoneNumber}
+          onChange={({ currentTarget: { value } }) => {
+            setUserDataField("phoneNumber", value);
+          }}
+        />
+        <Select
+          placeholder="Tip de relație"
+          options={options.relation}
+          selectProps={{
+            onChange: ({ currentTarget: { selectedIndex } }) => {
+              setUserDataField("relationType", selectedIndex);
+            }
+          }}
+        ></Select>
+        <Select
+          placeholder="Judet"
+          options={options.county}
+          selectProps={{
+            onChange: ({ currentTarget: { selectedIndex } }) => {
+              setUserDataField("county", selectedIndex);
+            }
+          }}
+        ></Select>
+        <Select
+          placeholder="Localitate"
+          options={options.city}
+          selectProps={{
+            onChange: ({ currentTarget: { selectedIndex } }) => {
+              setUserDataField("city", selectedIndex);
+            }
+          }}
+        ></Select>
+        <Input
```

Comment:
We now have a [number input type](https://taskforce-fe-components.now.sh/?path=/story/input--types) with `min` and `max`